spec/spec_helper.rb

```diff
@@ -18,5 +19,70 @@
 
   config.expect_with :rspec do |c|
     c.syntax = :expect
+    c.include_chain_clauses_in_custom_matcher_descriptions = true
+  end
+end
+
+RSpec::Matchers.define :parse_input do |input|
```

Comment:
I'll be honest here, these custom matchers are still a little bit beyond me, so I'm just going to trust that they do what they say on the tin.

Reply:
f115688